recipes-core/base-files/base-files_3.%.bbappend

```diff
@@ -1,8 +1,20 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+FILESEXTRAPATHS_prepend_menderimage := "${THISDIR}/${PN}:"
 
 #TODO: ${@bb.utils.contains("MACHINE_FEATURES", "sd", "packagegroup-mender-sd", "", d)}
 #./meta/recipes-core/packagegroups/packagegroup-base.bb
-do_install_append () {
+
+do_patch_device() {
```

Comment:
This is what we were discussing in the other ticket. The only way I could find to do conditionals in the bitbake recipe is via `OVERRIDES`. But since the task will be added to the run queue regardless of whether `menderimage` triggers or not, I needed an empty one to make it happy. But this is probably suboptimal, if you know a better way I'm all for changing it.